library/std/src/time.rs

```diff
@@ -368,7 +361,7 @@ impl Add<Duration> for Instant {
     /// This function may panic if the resulting point in time cannot be represented by the
     /// underlying data structure. See [`checked_add`] for a version without panic.
     ///
-    /// [`checked_add`]: ../../std/time/struct.Instant.html#method.checked_add
+    /// [`checked_add`]: Instant::checked_add
```

Comment:
We could also do the same trick here.

Reply:
This is already being merged so I won't modify further but a follow up PR would probably be accepted without any problem. 😃